addons/mail/static/src/core/common/suggestion_service.js

```diff
@@ -40,6 +40,7 @@ export class SuggestionService {
      * @param {import("models").Thread} [thread]
      */
     async fetchPartners(term, thread) {
+        this.env.services.im_status.fetch();
```

Comment:
Why calling it here? The RPC should return the values.

Reply:
Indeed, I thought we only retrieved unknown partners, removed.